### PR TITLE
Use importlib.metadata from the standard library on Python 3.8+

### LINKED ISSUE #1367: Use importlib.metadata from the standard library on Python 3.8
Hello. I'd like tox to use the standard library [importlib.metadata](https://docs.python.org/3.8/library/importlib.metadata.html) instead of the [importlib_metadata](https://pypi.org/project/importlib-metadata/) backport on Python 3.8.

See fox example https://github.com/pytest-dev/pluggy/pull/223 or https://github.com/pytest-dev/pytest/pull/5539

I'll work on a PR.

### PR DESCRIPTION
**Pull request by hroncok**
Resolves #1367

### CODE REVIEW

File: docs/changelog/1367.misc.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Replace ``importlib_metadata`` backport with ``importlib.metadata``
+from the standard library on Python 3.8+ - by :user:`hroncok`
```
Comment (gaborbernat) [inline docs/changelog/1367.misc.rst]: can you put here `3.8+` in `` ` ``?

Comment (hroncok) [reply docs/changelog/1367.misc.rst]: done

**Comment by hroncok** [comment]
> Can you pull in here 3.8 to test this?

Separate commit?

**Comment by gaborbernat** [comment]
Here is perfect 👍

**Comment by hroncok** [comment]
Azure Pipelines failed on various Windows. I don't think it is related.

**Comment by gaborbernat** [comment]
That's alright 👍

**Comment by gaborbernat** [comment]
Thanks!

**Comment by hroncok** [comment]
Thanks for the absolutely swift review!

**Pull request merged**